cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -53,6 +53,24 @@ EOF
   echo "will deprovision GCE cluster ${infraID} in region ${region}"
 done
 
+libvirt_p_cluster_age_cutoff="$(TZ=":America/New_York" date --date="-11 hours" '+%Y-%m-%dT%H:%M%z')"
+echo "deprovisioning clusters with creation time before ${libvirt_p_cluster_age_cutoff} in libvirt-p"
+for network in $(ssh ${bastion_p} virsh net-list --all --name | grep -v default) ; do
+        if [[ ! -z $(ssh ${bastion_p} journalctl -u libvirtd.service --until \"11 hours ago\" 2>/dev/null | grep ${network}) ]] ; then
```

Comment:
nit: we're using two spaces as indentation in the rest of the file, can we please follow that format

Reply:
oh yah, definitely meant to fix that. i was focusing more on big-picture "will this work" with this submission.